api/proto/wso2/discovery/api/api.proto

```diff
@@ -49,14 +49,15 @@ message Api {
 	repeated SecurityList security = 13;
 	EndpointSecurity endpointSecurity = 14;
 	string authorizationHeader = 15;
-	bool disableSecurity = 16;
-	string vhost = 17;
-	string organizationId = 18;
-	string apiProvider = 19;
-	bool enableBackendJWT = 20;
-	string deploymentType = 21;
-	string environmentId = 22;
-	string environmentName = 23;
-	BackendJWTConfiguration backendJWTConfiguration = 24;
-	ChoreoComponentInfo choreoComponentInfo = 25;
+	string apiKeyHeader = 16;
```

Comment:
Having both authorizationHeader and apiKeyHeader together is preferable. However, during a release that updates deployments, an older Enforcer might connect to a new Adapter, or vice versa. To ensure safety, let’s move it to the last number :)

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/3660/commits/92840d48345b37fe6957c523d56a16c81fe5758f